app/views/inputSingleCheckbox.scala.html

```diff
@@ -14,13 +14,14 @@
         @errorMessage(elements)
         <div class='govuk-checkboxes__item @if(elements.args.contains(Symbol("_smallCheckbox"))){govuk-checkboxes--small}'>
             <input
-                    @isSelected
-                    class="govuk-checkboxes__input"
-                    id="@elements.id"
-                    name="@elements.id"
-                    type="checkbox"
-                    value="@value"
-            />
+                @isSelected
+                class="govuk-checkboxes__input"
+                id="@elements.id"
+                name="@elements.id"
+                type="checkbox"
+                value="@value"
+                @elements.args.disabledInput(value.toString())
```

Comment:
What are your thoughts on keeping these `FormFunctions` methods, after our discussion yesterday? Are you still planning to come back and improve them, or did you decide that they're OK as they are?

Reply:
Yes, I was initially thinking of making the change now but then I'd have to change every place that's using it, so I thought, it's best to keep it as it is for now and then afterwards, create a new branch and start the refactoring there.